lesson8/lesson8.js

```diff
@@ -33,12 +33,14 @@ function createList(item){
 
 function fetchListData() {
   showLoadingImg();
-  return new Promise((resolve) => {
-  setTimeout(() => resolve(listItems), 3000)
-  });
+  return new Promise(function(resolve, reject) {
+    setTimeout(() => reject(Error("データを取得できませんでした")), 3000)
+    })
 }
 
-fetchListData().then((item) => {
+fetchListData()
+.then((listItems) => {
   deleteLoadingImg();
-  createList(item)
-});
+  console.log(createList(listItems))
+})
+.catch(console.error)
```

Comment:
【LGTM】
I thought it would be better to use console.error.

I searched for console.error.
If you just want to issue an error message, I think that "console.log" is fine, but if it is "console.error", a stack trace is output with the message, and I found out that it has the advantage of being easy to debug.

I am also learning about this task, so I will refer to it.

Reply:
The argument of catch puts the exception identifier
Write what you want to do in the scope of the catch block.

Reply:
The argument of catch puts the exception identifier.
Write what you want to do in the scope of the catch block.

Reply:
https://github.com/ja0112hk さん、調べたことを共有してくださりありがとうございます。勉強になりました。